Hymns.xcodeproj/project.pbxproj

```diff
@@ -816,7 +816,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"${PODS_ROOT}/Fabric/run\"\n";
+			shellScript = "\"${PODS_ROOT}/FirebaseCrashlytics/run\"\n";
```

Comment:
I se "/path/to/pods/directory/FirebaseCrashlytics/upload-symbols" in https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=ios#the_run_and_upload-symbols_scripts_are_now_in_firebasecrashlytics. Do you know what that's for?